appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.66.2",
+    "version": "0.67.0",
```

Comment:
I wasn't really sure what good/bad could happen if the consumer is using the older version of the `ui` package along with a newer version of this one--so I thought the safest bet is to rev the major on this one as well. Let me know what you think.

Reply:
Yeah this is safest